.github/workflows/pr-check.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: PR check
+on:
+  pull_request:
+jobs:
```

Comment:
maybe here we should have a matrix to build as well the rhel.Dockerfile ? so we know that this alternate Dockerfile works as well